app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,18 @@
+class CommentsController < ApplicationController
+  def create
+    @comment = current_user.comments.new(comment_params)
+    @comment.post_id = params[:post_id]
+
+    if @comment.save
+      redirect_to user_post_path(current_user, @comment.post)
+    else
+      render :create
+    end
+  end
```

Comment:
- [ ] It would be better if our comments controller has the 2 methods present, thus the `new` and `create` methods. The `new` method makes the `GET` request and the `create` method makes the `POST` requests

Reply:
thanks for your good and valuable feedback I apply your suggestion